libdedx/dedx.c

```diff
@@ -664,6 +664,24 @@ int _dedx_load_atima(stopping_data * data, dedx_config * config, float * energy,
 }
 
 
+float dedx_get_simple_stp_for_program(const int program, const int ion, const int target, float energy, int *err) {
+    dedx_config *config = (dedx_config *)calloc(1,sizeof(dedx_config));
+    float stp;
+    config->target = target;
+    config->ion = ion;
+    config->program = program;
+    dedx_workspace *ws = dedx_allocate_workspace(1,err);
+    if(*err != 0)
+        return 0;
```

Comment:
`-1` would be a better value to return in case of errors

Reply:
this is the same return value as in `dedx_get_stp`

Reply:
Then OK